src/Symfony/Component/Form/ChoiceList/Factory/CachingFactoryDecorator.php

```diff
@@ -174,14 +174,16 @@ public function createListFromLoader(ChoiceLoaderInterface $loader, $value = nul
     /**
      * {@inheritdoc}
      *
-     * @param array|callable|Cache\PreferredChoice|null $preferredChoices The preferred choices
-     * @param callable|false|Cache\ChoiceLabel|null     $label            The option or static option generating the choice labels
-     * @param callable|Cache\ChoiceFieldName|null       $index            The option or static option generating the view indices
-     * @param callable|Cache\GroupBy|null               $groupBy          The option or static option generating the group names
-     * @param array|callable|Cache\ChoiceAttr|null      $attr             The option or static option generating the HTML attributes
+     * @param array|callable|Cache\PreferredChoice|null        $preferredChoices           The preferred choices
+     * @param callable|false|Cache\ChoiceLabel|null            $label                      The option or static option generating the choice labels
+     * @param callable|Cache\ChoiceFieldName|null              $index                      The option or static option generating the view indices
+     * @param callable|Cache\GroupBy|null                      $groupBy                    The option or static option generating the group names
+     * @param array|callable|Cache\ChoiceAttr|null             $attr                       The option or static option generating the HTML attributes
+     * @param array|callable|Cache\ChoiceTranslationParameters $labelTranslationParameters The parameters used to translate the choice labels
      */
-    public function createView(ChoiceListInterface $list, $preferredChoices = null, $label = null, $index = null, $groupBy = null, $attr = null)
+    public function createView(ChoiceListInterface $list, $preferredChoices = null, $label = null, $index = null, $groupBy = null, $attr = null/*, $labelTranslationParameters = []*/)
     {
+        $labelTranslationParameters = \func_get_args()[6] ?? [];
```

Comment:
better use func_num_args

Reply:
You mean 
```
        if (\func_num_args() >= 6) {
            $labelTranslationParameters = \func_get_args()[6];
        } else {
            $labelTranslationParameters = [];
        }
```
?

Reply:
Yes. More exactly (as found elsewhere in the codebase):
`$labelTranslationParameters = 5 < \func_num_args() ? func_get_arg(5) : [];`

Reply:
(with the correct index/offset :)